### DISCUSSION TITLE
Feat: Pagination suggestion

### DISCUSSION THREAD

**Discussion by jopsa** [gokulkrishh/expense.fyi#97] [Ideas]
I'm a user of your project and I'm really enjoying it!

I was wondering if you have any plans to implement pagination in the tables where the main records of each model are listed. I think it would be a great addition to the project and would make it more user-friendly.

Thanks for your time.

Best regards,

**Comment by gokulkrishh** [comment]
@jopsa Glad you enjoying our app. I do have pagination feature in next milestone. Thanks for suggesting.
